### PR TITLE
feat: enable config callstats configParams

### PR DESCRIPTION
**Pull request by pangrr**
As part 2 of #1669

### CODE REVIEW

File: modules/statistics/CallStats.js [hunk 1]
```diff
@@ -361,14 +361,10 @@ export default class CallStats {
             CallStats.callStatsID = options.callStatsID;
             CallStats.callStatsSecret = options.callStatsSecret;
 
-            let configParams;
+            const configParams = {...options.configParams};
```
Comment (horymury) [inline modules/statistics/CallStats.js:364]: document the new options.configParams

File: modules/statistics/statistics.js [hunk 2]
```diff
@@ -72,7 +72,8 @@ function _initCallStatsBackend(options) {
         applicationName: options.applicationName,
         getWiFiStatsMethod: options.getWiFiStatsMethod,
         confID: options.confID,
-        siteID: options.siteID
+        siteID: options.siteID,
+        configParams: options.configParams
```
Comment (horymury) [inline modules/statistics/statistics.js:76]: document configParams in the Statistics function.

**Review by horymury — CHANGES_REQUESTED** [review]

File: modules/statistics/CallStats.js (hunk 1 above)
Comment (pangrr) [reply modules/statistics/CallStats.js:365]: commented

File: modules/statistics/statistics.js (hunk 2 above)
Comment (pangrr) [reply modules/statistics/statistics.js:77]: commented

**Comment by horymury** [comment]
@pangrr sorry I meant add JSDocs param description on the  `static initBackend(options)` and `export default function Statistics(xmpp, options)` functions :)

**Comment by pangrr** [comment]
> @pangrr sorry I meant add JSDocs param description on the `static initBackend(options)` and `export default function Statistics(xmpp, options)` functions :)

Done. Thanks for explaining.

**Review by horymury — APPROVED** [review]

**Pull request merged**